Dashboard/test_load.py

```diff
@@ -0,0 +1,71 @@
+from unittest.mock import MagicMock, patch
+
+import pytest
+from psycopg2.extensions import connection, cursor
+
+
+from load import (
+    verify_keys,
+    insert_product_information
+)
```

Comment:
```suggestion
from unittest.mock import MagicMock, patch
import pytest

from psycopg2.extensions import connection, cursor

from load import  verify_keys, insert_product_information
```

Reply:
I prefer importing like this in test files because if that file is expanded, it explicitly reminds me to place the imports on a new line!